gui-requirements.txt

```diff
@@ -0,0 +1,2 @@
+PyQt5
```

Comment:
There's at least some potential consideration for not having PyQt5 here, I don't remember best practices for qtpy apps

Reply:
.sWe include `PyQt5` for both `atef` and `lightpath`, which I was looking at for reference.  You're right that `qtpy` is supposed to be our qt-agnostic translation layer, though without it pip won't install PyQt5.  (at least from what I can verify from my testing)